ChangeLog.md

```diff
@@ -21,7 +21,7 @@ All deprecated APIs will be removed in the next release.
 
 ## Other changes
 
-- Adds `Monad` instances for all carrier types.
```

Comment:
@patrickt: Looks like we dropped the `Monad` instances for all carrier types line by accident.